docs/cookbook/getParseableZonedStringAtInstant.mjs

```diff
@@ -1,19 +1,19 @@
-const instantTime = Temporal.Instant.from('2020-01-03T10:41:51Z');
+const exactTime = Temporal.Instant.from('2020-01-03T10:41:51Z');
```

Comment:
"Exact" may be the adjective, but "instant" is the noun. So I'd probably call this variable `instant` (not `exactTime`) to more clearly associate it with the type used.